plugins/modules/win_dns_record.py

```diff
@@ -19,6 +19,20 @@
     - The name of the record.
     required: yes
     type: str
+  port:
+    description:
+    - The port number of the record.
+    - Optional for C(type=SRV).
```

Comment:
It doesn't seem to be optional, the docs for `Add-DnsServerResourceRecord` show it as a mandatory parameter.,

Reply:
Yes, as I have told you above we are not enforcing the user to provide these values instead we are taking it as default.